File: src/test/java/selenium/tests/LoginPage.java
package selenium.tests;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

/**
 * Created by Administrator on 2016-09-23.
 */
public class LoginPage extends Abstract{
    public LoginPage(WebDriver driver) {
        super(driver);
    }

    public AdminPage logIn() {
        driver.get(baseUrl + "/wp-login.php");
        driver.findElement(By.id("user_login")).clear();
        driver.findElement(By.id("user_login")).sendKeys("szkolenieautomatyzacja");
        driver.findElement(By.id("user_pass")).clear();
        driver.findElement(By.id("user_pass")).sendKeys("QW12qw12");
        driver.findElement(By.id("wp-submit")).click();
        return new AdminPage(driver);
    }
}

File: testautomation/testautomation/Test/LoginPage.cs
using System;
using System.Text;
using System.Text.RegularExpressions;
using System.Threading;
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Firefox;
using OpenQA.Selenium.Chrome;
using OpenQA.Selenium.Support.UI;

namespace PageObjectPattern
{
    internal class LoginPage :Page
    {
        public LoginPage(IWebDriver driver) : base(driver)
        {
        }

        internal AdminPage LogIn(string v1, string v2, String baseURL)
        {
            driver.Navigate().GoToUrl(baseURL + "/wp-admin");
            driver.FindElement(By.Id("user_login")).Clear();
            driver.FindElement(By.Id("user_login")).SendKeys(v1);
            driver.FindElement(By.Id("user_pass")).Clear();
            driver.FindElement(By.Id("user_pass")).SendKeys(v2);
            driver.FindElement(By.Id("wp-submit")).Click();
            return new AdminPage(driver);
        }
    }
}